src/components/feature/settings/FileManagement/index.tsx

```
@@ -0,0 +1,63 @@
+      }
+    };
+
+    fetchFiles();
+  }, []);
+
+  const handleDelete = async (fileId: string) => {
+    try {
+      await deleteFile(fileId);
+      setFiles((prevFiles) => prevFiles.filter((file) => file.id !== fileId));
+      toast.success("ファイルを削除しました");
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)